central/graphql/resolvers/node_components.go

```diff
@@ -97,11 +132,151 @@ func (resolver *Resolver) NodeComponentCount(ctx context.Context, args RawQuery)
 
 		return resolver.componentCountV2(ctx, RawQuery{Query: &query})
 	}
-	// TODO : Add postgres support
-	return 0, errors.New("Resolver NodeComponentCount does not support postgres yet")
+
+	if err := readNodes(ctx); err != nil {
+		return 0, err
+	}
+	query, err := args.AsV1QueryOrEmpty()
+	if err != nil {
+		return 0, err
+	}
+	loader, err := loaders.GetNodeComponentLoader(ctx)
+	if err != nil {
+		return 0, err
+	}
+	query = tryUnsuppressedQuery(query)
+
+	return loader.CountFromQuery(ctx, query)
 }
 
+/*
+Utility Functions
+*/
+
 func queryWithNodeIDRegexFilter(q string) string {
 	return search.AddRawQueriesAsConjunction(q,
 		search.NewQueryBuilder().AddRegexes(search.NodeID, ".+").Query())
 }
+
+func (resolver *nodeComponentResolver) withNodeComponentScope(ctx context.Context) context.Context {
+	return scoped.Context(ctx, scoped.Scope{
+		Level: v1.SearchCategory_NODE_COMPONENTS,
+		ID:    resolver.data.GetId(),
+	})
+}
+
+func (resolver *nodeComponentResolver) nodeComponentQuery() *v1.Query {
+	return search.NewQueryBuilder().AddExactMatches(search.ComponentID, resolver.data.GetId()).ProtoQuery()
+}
+
+func (resolver *nodeComponentResolver) nodeComponentRawQuery() string {
+	return search.NewQueryBuilder().AddExactMatches(search.ComponentID, resolver.data.GetId()).Query()
+}
+
+/*
+Sub Resolver Functions
+*/
+
+// FixedIn returns the node component version that fixes all the fixable vulnerabilities in this component.
+func (resolver *nodeComponentResolver) FixedIn(ctx context.Context) string {
+	defer metrics.SetGraphQLOperationDurationTime(time.Now(), pkgMetrics.NodeComponents, "FixedIn")
```

Comment:
As discussed offline, return empty for now since we never actually supported this field. We do not have support from the scanner. This field indicates the highest version of the component that fixes all the fixable vulnerabilities in node. More importantly, sort the version lexically does not give the highest version since each distro has a specific versioning scheme.